client/flutter/lib/pages/main_pages/check_up_poster_page.dart

```diff
@@ -0,0 +1,190 @@
+            child: _buildCardContent(context,
+                titleStyle: TextStyle(color: Constants.neutralTextLightColor)))
+        : Padding(
+            padding: const EdgeInsets.symmetric(vertical: 8.0),
+            child: ClipRRect(
+              borderRadius: BorderRadius.circular(8.0),
+              child: Container(
+                  color: CupertinoColors.white,
+                  padding: EdgeInsets.fromLTRB(12, 18, 0, 18),
+                  child: Row(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: <Widget>[
+                      Container(
+                        alignment: Alignment.center,
+                        child: SvgPicture.asset(
+                          this.assetName,
+                          color: iconColor,
+                          width: iconSize,
+                          height: iconSize,
+                        ),
+                      ),
+                      SizedBox(
+                        // its 12, but 2 is in the card content due to how Html widget works
+                        width: 10,
+                      ),
+                      Flexible(
+                        child: _buildCardContent(context),
+                      ),
+                    ],
+                  )),
+            ),
+          );
+  }
+
+  Column _buildCardContent(BuildContext context,
+      {TextStyle titleStyle = const TextStyle(fontWeight: FontWeight.w700)}) {
+    final defaultTextStyle = ThemedText.htmlStyleForVariant(
+        TypographyVariant.body,
+        textScaleFactor: MediaQuery.textScaleFactorOf(context));
+    final boldTextStyle =
+        defaultTextStyle.copyWith(fontWeight: FontWeight.w700);
+    ;
+    final title = ThemedText(
+      content.title,
+      variant: TypographyVariant.body,
+      style: titleStyle,
+    );
+    final child = title;
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      mainAxisAlignment: MainAxisAlignment.start,
+      children: <Widget>[
+        Padding(
+          // Html widget has 2 un-modifiable pixel padding, sigh...
+          padding: EdgeInsets.only(left: 2, right: 12),
+          child: child,
+        ),
+        if (content.bodyHtml != null)
+          Padding(
+            // Html widget has 2 un-modifiable pixel padding, sigh...
+            padding: EdgeInsets.only(right: 12, bottom: 0),
+            child: Html(
+              customEdgeInsets: (_) => EdgeInsets.zero,
+              data: content.bodyHtml,
+              defaultTextStyle: defaultTextStyle,
+              customTextStyle: (dom.Node node, TextStyle baseStyle) {
+                if (node is dom.Element) {
+                  switch (node.localName) {
+                    case 'b':
+                      return baseStyle.merge(boldTextStyle);
+                  }
+                }
+                return baseStyle.merge(defaultTextStyle);
```

Comment:
The nested `return` statements took me a while to parse. I think it would read better to move parts of this before the initial return statement.... particularly L184..173... then also line 168 onwards.

Reply:
Done